### DISCUSSION TITLE
Requesting new release due to vulnerability in v1.7.12

### DISCUSSION THREAD

**Discussion by kbessas** [Checkmarx/kics#6914] [General] [answered]
Buildkit had vulnerability CVE-2024-23652 that was fixed in version v0.12.5. That has already been merged with https://github.com/Checkmarx/kics/pull/6912 but a new release would also be nice for this to get a new docker image asap.

**Comment by gabriel-cx** [comment]
Hi @kbessas ,

Yes, a new KICS version (v1.7.13) is planned to be released this week.

**Comment by gabriel-cx** [comment, marked as answer]
Hi @kbessas ,

You can now use [KICS new version](https://github.com/Checkmarx/kics/releases/tag/v1.7.13)! :D
